scripts/create-package/package-template/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "PACKAGE_NAME",
+  "version": "0.0.0",
+  "description": "PACKAGE_DESCRIPTION",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/PACKAGE_DIRECTORY_NAME#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh PACKAGE_NAME",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest --reporters=jest-silent-reporter",
+    "test:clean": "jest --clearCache",
+    "test:verbose": "jest --verbose",
+    "test:watch": "jest --watch"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.4.3",
+    "@types/jest": "^27.4.1",
+    "deepmerge": "^4.2.2",
+    "jest": "^27.5.1",
+    "ts-jest": "^27.1.4",
```

Can the `jest` deps in non-root packages be removed now that they're installed at the root level?

```suggestion
    "deepmerge": "^4.2.2",
```

---

Good question. As it turns out, that doesn't work:
```
yarn workspace @metamask/foo run test
command not found: jest
```

---

Good catch! Doesn't seem like there should be any issue with including these packages as devDeps in any case.